src/pages/loading/FullPageSpinner.tsx

```diff
@@ -0,0 +1,34 @@
+import { Theme } from '@material-ui/core';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import { makeStyles } from '@material-ui/styles';
+import { CSSProperties } from '@material-ui/styles/withStyles';
+import React from 'react';
+import { FCProps } from 'src/shared/types/FCProps';
+import { Styles } from 'src/shared/types/Styles';
+
+interface OwnProps {}
+
+type Props = FCProps<OwnProps> & StyleProps;
+
+export function FullPageSpinner(props: Props) {
+  const classes = useStyles(props);
+  return (
+    <div className={classes.root}>
```

Comment:
styled component?

Reply:
more than one element is in this component, so useStyle is better